lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -45,10 +45,14 @@ export const BrowserAuthErrorMessage = {
         code: "user_cancelled",
         desc: "User cancelled the flow."
     },
-    monitorWindowTimeoutError: {
+    monitorPopupTimeoutError: {
         code: "monitor_window_timeout",
         desc: "Token acquisition in popup failed due to timeout."
     },
+    monitorIframeTimeoutError: {
+        code: "monitor_window_timeout",
```

Comment:
Keeping error code the same as there are likely apps relying on this error code for certain error handling logic.